tests/method-chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -172,39 +172,58 @@ function f() {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 export default function theFunction(action$, store) {
   return action$.ofType(THE_ACTION).switchMap(action => Observable.webSocket({
-      url: THE_URL,
-      more: stuff(),
-      evenMore: stuff({
-        value1: true,
-        value2: false,
-        value3: false
-      })
+    url: THE_URL,
+    more: stuff(),
+    evenMore: stuff({
+      value1: true,
+      value2: false,
+      value3: false
     })
+  })
     .filter(data => theFilter(data))
     .map(({ theType, ...data }) => theMap(theType, data))
     .retryWhen(errors => errors));
 }
 
 function f() {
   return this._getWorker(workerOptions)({
-      filePath,
-      hasteImplModulePath: this._options.hasteImplModulePath
-    })
-    .then(metadata => {
-      // \`1\` for truthy values instead of \`true\` to save cache space.
-      fileMetadata[H.VISITED] = 1;
-      const metadataId = metadata.id;
-      const metadataModule = metadata.module;
-      if (metadataId && metadataModule) {
-        fileMetadata[H.ID] = metadataId;
-        setModule(metadataId, metadataModule);
-      }
-      fileMetadata[H.DEPENDENCIES] = metadata.dependencies || [];
-    });
+    filePath,
+    hasteImplModulePath: this._options.hasteImplModulePath
+  }).then(metadata => {
+    // \`1\` for truthy values instead of \`true\` to save cache space.
+    fileMetadata[H.VISITED] = 1;
+    const metadataId = metadata.id;
+    const metadataModule = metadata.module;
+    if (metadataId && metadataModule) {
+      fileMetadata[H.ID] = metadataId;
+      setModule(metadataId, metadataModule);
+    }
+    fileMetadata[H.DEPENDENCIES] = metadata.dependencies || [];
+  });
 }
 "
 `;
 
+exports[`inline_merge.js 1`] = `
+"Object.keys(
+  availableLocales({
+    test: true
+  })
+)
+.forEach(locale => {
+  // ...
+});
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Object.keys(
+  availableLocales({
+    test: true
+  })
+).forEach(locale => {
```

Comment:
FYI, we already implement this style in the case

```js
call(
  ...
).call(
  ...
)
```

but it didn't work because of this "bug"